ui/src/tool-panel/location-track/location-track-location-infobox.tsx

```diff
@@ -223,6 +238,30 @@ export const LocationTrackLocationInfobox: React.FC<LocationTrackLocationInfobox
                             },
                         };
                     });
+                    const startSwitchId =
+                        extraInfo.startSplitPoint.type === 'SWITCH_SPLIT_POINT'
+                            ? extraInfo.startSplitPoint.switchId
+                            : undefined;
+                    const endSwitchId =
+                        extraInfo.endSplitPoint.type === 'SWITCH_SPLIT_POINT'
+                            ? extraInfo.endSplitPoint.switchId
+                            : undefined;
+
+                    const filterUniqueSwitchSplitPoint = filterUniqueById(
+                        (sp: SwitchSplitPoint) => sp.switchId,
+                    );
```

Comment:
Olin jo tekemässä tätä tiedoston sisäiseksi globaaliksi funktioksi `isSplittablePoint`:in rinnalle, ideana että se söisi pelkän `SplitPoint`:in ja että sitä voisi käyttää osana tuota `isSplittablePoint`-tarkistelua, mutta ei mennytkään niinkuin elokuvissa. `filterUniqueById` pitää sisäisesti kirjaa näkemistään objekteista, ja siten jos siitä tekee tuollaisen globaalin version, niin se muistaa ikuisesti kaiken aiemmin näkemänsä -> se antaisi eri tulokset samalle datalle useampaan kertaan ajettuna. Tuo tuskin on toivottua. Olisin korjannut tämän, mutta sen toteutus jotenkin muuten olisi vaatinut mietintää ja mulla alkaa käymään se vähiin ennen lomaa -> teen tämän korjaamisesta oman tikettinsä.